aws/lambda/nodemanager.py

```diff
@@ -30,7 +30,13 @@ def eni_wait_for_attachment(eni_id):
             retries += 1
             continue
 
-        status = eni_info["Attachment"]["Status"]
+        if not eni_info.attachment:
+            log.warning(
```

Comment:
We don't log out warning or error when we are waiting for a status (detachment, attachment, etc.), otherwise it will become very overwhelming. 

Also, we should sleep before we retry. See following lines as an example.